db/migrate/20250128164211_add_missing_reading_window_to_amr_data_feed_config.rb

```diff
@@ -0,0 +1,6 @@
+class AddMissingReadingWindowToAmrDataFeedConfig < ActiveRecord::Migration[7.1]
+  def change
+    add_column :amr_data_feed_configs, :missing_reading_window, :integer
+    change_column_default :amr_data_feed_configs, :missing_reading_window, from: nil, to: 5
```

Comment:
Can you not just add `null: true, default: 5` to the `add_column` statement?

Reply:
am thinking it's better to set all the existing ones to nil and then have 5 as the default for any new ones?